### PR TITLE
Initial Development: Generate ts file as QML file localization output

### PR DESCRIPTION
**Pull request by gouniLee**
As an output of qml file parsing. It generates ts file which is xml form.
It should run with https://github.com/iLib-js/ilib-loctool-webos-qml module.

### CODE REVIEW

File: TSResourceFile.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+    return PrettyData.xml(xml);
+};
+
+/**
+ * Find the path for the resource file for the given project, context,
+ * and locale.
+ *
+ * @param {String} locale the name of the locale in which the resource
+ * file will reside
+ * @param {String|undefined} flavor the name of the flavor if any
```
[the remark this answers is not in the document]
Comment (gouniLee) [reply TSResourceFile.js]: I updated it.

File: TSResourceFile.js [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+    return str.replace(/\s+/, " ").trim();
+}
+
+/**
+ * @private
+ */
+TSResourceFile.prototype.getDefaultSpec = function() {
+    if (!this.defaultSpec) {
+        this.defaultSpec = this.project.settings.localeDefaults ?
+            this.API.utils.getLocaleDefault(this.locale, this.flavor, this.project.settings.localeDefaults) :
```
Comment (ehoogerbeets) [inline TSResourceFile.js]: There is no "this.flavor" in a `TSResourceFile` right?

Comment (gouniLee) [reply TSResourceFile.js]: Yes, `this.flavor` is always `undefined`. I've looked then I realize `getDefaultSpec` is not using anywhere. I think it's better to remove the whole function here.

Comment (ehoogerbeets) [reply TSResourceFile.js]: yes, please remove

**Pull request merged**